source/GlassView.Benchmark/ComplexBenchmark.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System.Numerics;
+using BenchmarkDotNet.Attributes;
+
+namespace Atmoos.GlassView.Benchmark;
+
+/* This is only a dummy benchmark for us to play around with exports. */
+
+[ShortRunJob]
+[IterationCount(7)]
+[MemoryDiagnoser]
+[BenchmarkCategory("Complex")]
+public class ComplexBenchmark
+{
+    private Double[][] left, right;
+    [Params(54, 42)]
+    public Int32 Rows { get; set; }
+    [Params(12, 34, 123)]
+    public Int32 Center { get; set; }
+
+    [Params(21, 76)]
+    public Int32 Cols { get; set; }
+
+    [GlobalSetup]
+    public void Setup()
+    {
+        this.left = Matrix(Rows, Center);
+        // As they're arrays, the dimensions looks the wrong
+        // way round on the right hand side matrix...
+        this.right = Matrix(Cols, Center);
+    }
+
+
+    [Benchmark(Baseline = true), BenchmarkCategory("Vectorised")]
+    public Object VectorisedMultiplication() => MatrixProduct(this.left, this.right, VectorisedDotProduct);
+
+    [Benchmark, BenchmarkCategory("Regular")]
+    public Object RegularMultiplication() => MatrixProduct(this.left, this.right, DotProduct);
+
+    private static Double[] Vec(Int32 count)
+        => Enumerable.Range(0, count).Select(i => i - count / 2d).ToArray();
+    private static Double[][] Matrix(Int32 rows, Int32 cols)
+        => Enumerable.Range(0, rows).Select(_ => Vec(cols)).ToArray();
+
+
+    public static Double DotProduct(Double[] left, Double[] right)
+    {
+        var sum = 0d;
```

Comment:
```suggestion
        Double sum = 0d;
```
Personally, I'd use var only for complex types and the System types for cases like String, Boolean, Double. However, we can discuss about this and write an editorconfig rule with whatever we decide :)

Reply:
Let's take this to issue #14 and then run `dotnet format` once.

Reply:
I did take this comment as a hint and incentive to get rid of the ugly and repetitive `File.ReadAllText()` call...

Reply:
Will be solved using `dotnet format`.